Move the product quantity selector to a different line of product price to better visualization

paul0vinicius opened the pull request:
#### What is the purpose of this pull request?
As the title describes.
<!--- Describe your changes in detail. -->

#### What problem is this solving?
<!--- What is the motivation and context for this change? -->
The overlapping of information in cases that the price is too high.

#### How should this be manually tested?
[Workspace Link](https://minicartpricealignment--storecomponents.myvtex.com/)

#### Screenshots or example usage
Before

Mobile
![captura de tela de 2019-01-31 17-48-50](https://user-images.githubusercontent.com/9946330/52084316-90fc3480-2580-11e9-8232-eb397e0924f2.png)

Desktop
![captura de tela de 2019-01-31 17-51-52](https://user-images.githubusercontent.com/9946330/52084525-0a942280-2581-11e9-96ae-53e50962416d.png)


After

Mobile
![captura de tela de 2019-01-31 17-50-13](https://user-images.githubusercontent.com/9946330/52084381-bdb04c00-2580-11e9-96b3-1cfcfae67997.png)

Desktop
![captura de tela de 2019-01-31 17-52-46](https://user-images.githubusercontent.com/9946330/52084572-226ba680-2581-11e9-8584-7a9bb009475a.png)

#### Types of changes
- [x] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

paul0vinicius commented:
> The bottom border of the item is "sambing" when the quantity and price is being updated. Maybe you could put a fixed value for the price div to avoid it.

Solved. For now I am waiting the suggestion by @oldufre to avoid the spinner.

theomoura commented:
There is another problem now, the entire minicart layout flicks when the quantity and price is being updated (web mode).

paul0vinicius commented:
> There is another problem now, the entire minicart layout flicks when the quantity and price is being updated (web mode).

This problem was also present in the master branch. I opened an [issue](https://github.com/vtex-apps/product-summary/issues/107) about it.

paul0vinicius commented:
New _bug_ detected! It is still under investigation.

![image](https://user-images.githubusercontent.com/9946330/52138109-06244400-262b-11e9-9c3a-206eb9a184d8.png)

jgfidelis commented:
<img width="326" alt="screen shot 2019-02-07 at 12 30 30 pm" src="https://user-images.githubusercontent.com/4925068/52417985-31e47580-2ad4-11e9-8ac9-344ce99e522a.png">

not right

jgfidelis commented:
Eu acho que tudo foi se baseando numa versão em que o botão de + ou - estava muito grande, o que foi corrigido. Um print atual da versão mobile:
<img width="259" alt="screen shot 2019-02-07 at 12 48 05 pm" src="https://user-images.githubusercontent.com/4925068/52419143-a7e9dc00-2ad6-11e9-8842-7a4af10aa212.png">


Está bem melhor. A única correção necessária é fazer tudo ficar um pouco abaixo para não ter overlap com o ícone de remoção.

Por favor, conversar com nosso desginer @galc se realmente quiserem seguir em frente com esse layout. Acho o original muito melhor.

paul0vinicius commented:
> <img alt="screen shot 2019-02-07 at 12 30 30 pm" width="326" src="https://user-images.githubusercontent.com/4925068/52417985-31e47580-2ad4-11e9-8ac9-344ce99e522a.png">
> 
> not right

Fixed. You can test it [here](https://paulo--phmex.myvtex.com/).

![captura de tela de 2019-02-07 15-34-39](https://user-images.githubusercontent.com/9946330/52434395-f6ef3b80-2aed-11e9-8506-a49a51e94e88.png)



> see image in comment
> 
> There are also a bunch o unsued variables after this changes. Remove them.

Also fixed. There were some pieces of unused code after refactoring this app.



> Eu acho que tudo foi se baseando numa versão em que o botão de + ou - estava muito grande, o que foi corrigido. Um print atual da versão mobile:
> <img alt="screen shot 2019-02-07 at 12 48 05 pm" width="259" src="https://user-images.githubusercontent.com/4925068/52419143-a7e9dc00-2ad6-11e9-8842-7a4af10aa212.png">
> 
> Está bem melhor. A única correção necessária é fazer tudo ficar um pouco abaixo para não ter overlap com o ícone de remoção.
> 
> Por favor, conversar com nosso desginer @galc se realmente quiserem seguir em frente com esse layout. Acho o original muito melhor.

Pâmela e Galc conversaram sobre o design e chegaram a conclusão de que quando os preços são muito grandes, o _overlapping_ acontece de qualquer forma. Sendo assim, é mais interessante definir uma linha para cada elemento do minicart, de modo que o preço pode crescer indefinidamente sem interferir em outro elemento.
